Update Hat Sample Pack Link + Added Mods Checklist (#46)
DesktopGooseUnofficial/ResourceHub: README.md

### Okay I got a mod, placed it in the right folder and all that but it's not working, how do I enable it?

![](https://i.imgur.com/ZNB4wsy.gif)

> ### If your mod isn't working please look at the check list to see what  step might be missing:

Reviewer: Why are there random spaces here? Here's a fix:
```suggestion
### If your mod isn't working please look at the checklist to see what step might be missing:
```

> **Check list:**

Reviewer: Again, why is there a space here?
```suggestion
**Checklist:**
```

[   ] Latest version of Desktop Goose

> [   ] mod dll is in mods folder within its own folder

Reviewer: ```suggestion
[   ] Mod DLL is in mods folder within its own folder
```

> [   ] config file has EnableMods=True

Reviewer: ```suggestion
[   ] Config file has EnableMods=True
```
